packages/slice-machine/lib/models/common/tracker.ts

```diff
@@ -1,38 +1,93 @@
 import Analytics from "analytics-node";
```

Comment:
I don't think this file should be in the lib as it's only a backend service, we should put it in the server/src/api/services instead. Only stuff shared between the front and the back should be added to this folder.